lua/nvim-web-devicons/icons-default.lua

```diff
@@ -287,6 +299,12 @@ local icons_by_filename = {
     cterm_color = "185",
     name = "License",
   },
+  ["init.lua"] = {
```

Comment:
Sorry, we can't take this one: `init.lua` is the lua standard for the default module in a directory.

neovim is just following that standard.

Reply:
oh ok ill remove that